index.html

```diff
@@ -49,9 +49,9 @@
 						<a href="http://docs.astropy.org">Documentation</a>
 						<div class="dropdown-content">
 							<ul>
-								<li><a href="http://docs.astropy.org" target="_blank">Current Release</a></li>
+								<li><a href="http://docs.astropy.org" target="_blank">Current Release (3.1)</a></li>
```

Comment:
I'm -1 on this

Reply:
I think I know why, but can you explain?